common/test/acceptance/pages/lms/dashboard.py

```diff
@@ -177,3 +177,16 @@ def pre_requisite_message_displayed(self):
         Verify if pre-requisite course messages are being displayed.
         """
         return self.q(css='section.prerequisites > .tip').visible
+
+    def click_username_dropdown(self):
+        """Click username dropdown"""
+        self.q(css='.dropdown').first.click()
+
+    @property
+    def username_dropdown_links(self):
+        """Return list username dropdown links"""
```

Comment:
There has been some movement to use the Google standard for Python docstrings where the first line is on the same line as the opening ``"""``. I suppose we should make this a formal decision and start being consistent about it.

Here are some good examples that we've been following:
http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html